### PR TITLE
nasm 2.15.02

### PR DESCRIPTION
**Pull request by slhck**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by slhck** [comment]
This should additionally fix the original error reported in https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/issues/30.

File: Formula/nasm.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nasm < Formula
   desc "Netwide Assembler (NASM) is an 80x86 assembler"
   homepage "https://www.nasm.us/"
-  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.01/nasm-2.15.01.tar.xz"
-  sha256 "28a50f80d2f4023e444b113e9ddc57fcec2b2f295a07ce158cf3f18740375831"
+  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02-xdoc.tar.xz"
+  sha256 "a2ec81aeb86b61dee8e43bdc561449119a65fb429efa48abdc569dc7471b1981"
```
Comment (bayandin) [inline Formula/nasm.rb:5]: `https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02-xdoc.tar.xz` — it's documentation, the source itself:

```suggestion
  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02.tar.xz"
  sha256 "f4fd1329b1713e1ccd34b2fc121c4bcd278c9f91cc4cb205ae8fcd2e4728dd14"
```

Comment (slhck) [reply Formula/nasm.rb:5]: Oh, thanks for spotting this 🤦

**Comment by slhck** [comment]
@bayandin I don't know why the tests are failing; it seems a bit obscure to me. I just squashed your suggestion on top of my original commit and force-pushed, but something is up with the repo in the CI environment.

**Comment by Rylan12** [comment]
@slhck Thanks for the PR! This was due to a CI issue that has been resolved. I'll rerun the tests.

**Review by chenrui333 — APPROVED** [review]

**Comment by slhck** [comment]
> @slhck Thanks for the PR! This was due to a CI issue that has been resolved. I'll rerun the tests.
> 
> Edit: looks like someone else already triggered a rerun 😄

Thanks!

But I get some issues installing it:

```
➜ rm -rf $(brew --cache)

homebrew-ffmpeg/Formula on  revert-yasm via 💎 v2.6.6
➜ brew upgrade nasm
==> Upgrading 1 outdated package:
nasm 2.15.01 -> 2.15.02
==> Upgrading nasm 2.15.01 -> 2.15.02
==> Downloading https://homebrew.bintray.com/bottles/nasm-2.15.02.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/83/833f4b88e5d32727234e7934d77eebafa661b868bc19dc320fa032ab72e8419a?__gda__=exp=1593704559~hmac=7d773b4240ffc509
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: f9cfc8ad4a4a6e671da2ab8fd12ea19faac331bc427b54b16f5ece9e29a50843
  Actual: 833f4b88e5d32727234e7934d77eebafa661b868bc19dc320fa032ab72e8419a
 Archive: /Users/werner/Library/Caches/Homebrew/downloads/3705f5f0c6917e7ab5e95b790bd7a2f7e92374d159fccd7da1593d4147ac20ab--nasm-2.15.02.mojave.bottle.tar.gz
To retry an incomplete download, remove the file above.
Warning: Bottle installation failed: building from source.
==> Downloading https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02.tar.xz
```

**Comment by chenrui333** [comment]
@slhck I just did a rerun in my Mojave machine, seems working fine for me?

```
==> Downloading https://homebrew.bintray.com/bottles/nasm-2.15.02.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/83/833f4b88e5d32727234e7934d77eebafa661b868bc19dc320fa032ab7
######################################################################## 100.0%
==> Pouring nasm-2.15.02.mojave.bottle.tar.gz
🍺  /usr/local/Cellar/nasm/2.15.02: 29 files, 2.9MB
```

**Comment by chenrui333** [comment]
@slhck can you do a `brew reinstall nasm` to see if you still have the same issue?

**Comment by retokromer** [comment]
It’s working fine on macOS 10.14.6 here.

**Comment by Rylan12** [comment]
@slhck it looks like you're still on the branch for this PR. Make sure you switch back to `master` and run `brew update` to get the updated `bottle` block.

The `bottle` block is updated in a commit directly to `master` just after commit that closes this PR (in this case, the commit that updates the `bottle` block is https://github.com/Homebrew/homebrew-core/commit/e0abfb2567bf86cd381f8db527c2b9866e06b70f).

**Comment by slhck** [comment]
@Rylan12 You're right, that was it. Thank you!

**Pull request closed without merging**